tools/app_caches.py

```diff
@@ -9,8 +9,11 @@
 
 import tqdm
 
-from appslib.utils import (REPO_APPS_ROOT,  # pylint: disable=import-error
-                           get_catalog, git_repo_age)
+from appslib.utils import (
+    REPO_APPS_ROOT,  # pylint: disable=import-error
+    get_catalog,
+    git_repo_age,
+)
```

Comment:
Yeah that will break pylint

Reply:
We probably want to drop pylint in favor of either flake8 or ruff

Reply:
Agreed… but the issue is the same, linters don't like imports after custom code.